docs/source/checks/tabular/model_evaluation/plot_calibration_score.py

```diff
@@ -4,9 +4,30 @@
 
 Calibration Score
 *****************
+This notebook provides an overview for using and understanding the Calibration Score check.
+
+**Structure:**
+
+* `What is the Calibration Score check? <#what-is-the-calibration-score-check>`__
+* `Binary Classification <#binary-classification>`__
+* `Multi-class classification <#multi-class-classification>`__
+
+
+What is the Calibration Score check?
+==================================================
+The ``CalibrationScore`` check calculates the calibration curve with brier score for each class.
+Calibration curves (also known as reliability diagrams) compare how well the
+probabilistic predictions of a binary classifier are calibrated. It plots the true
+frequency of the positive label against its predicted probability, for binned predictions.
+The Brier score metric may be used to assess how well a classifier is calibrated
+(`Brier score <https://en.wikipedia.org/wiki/Brier_score>`_).
```

Comment:
Great! However, calibration can also be done on multiclass labels, by checking a one-vs-all method (mainly, just checking the proba for each class separately). Can you edit the text to reflect that as well?

Reply:
i copy pasted  from the calibration_score check itself.
so maybe change there also?

Reply:
I agree, the message there should be changed a bit as well